paperless.conf.example

```diff
@@ -102,3 +102,13 @@ PAPERLESS_SHARED_SECRET=""
 # as is "example.com,www.example.com", but NOT " example.com" or "example.com,"
 #PAPERLESS_ALLOWED_HOSTS="example.com,www.example.com"
 
+# Override the default UTC time zone here
+#PAPERLESS_TIME_ZONE=America/Los_Angeles
```

Comment:
I love the idea of making the timezone configurable, but the default is UTC, so that should be what we have here, commented out so the user knows their starting position.